ui/components/app/user-preferenced-currency-input/user-preferenced-currency-input.component.js

```diff
@@ -2,27 +2,21 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import CurrencyInput from '../currency-input';
 
+// Noticed this component is not used in codebase;
```

Comment:
Can you explain this comment?  If this component it not used, can it be removed entirely?

Reply:
tbh I was hesitant to remove the files totally; so i added the comment in case we get back to it for any reason